test/regress.c

```diff
@@ -145,7 +146,8 @@ test(char const * path, int limit)
     pkt_additional = NULL;
     nestegg_packet_additional_data(pkt, 1, &pkt_additional, &pkt_additional_length);
 
-    printf("%u %llu %u", pkt_track, (unsigned long long) pkt_tstamp, pkt_cnt);
+    printf("%d", pkt_keyframe_status);
+    printf(" %u %llu %u", pkt_track, (unsigned long long) pkt_tstamp, pkt_cnt);
```

Comment:
These two lines can be a single printf.  Move the keyframe result after the track number.